rs.ruffle.Ruffle.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: rs.ruffle.Ruffle
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: ruffle
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/openjdk17/bin
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: ruffle
+    buildsystem: simple
+    build-options:
+      env:
+        CARGO_HOME: /run/build/ruffle/cargo
+    build-commands:
+      - java --version
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --verbose
+      - install -D -T ./desktop/assets/icon.svg /app/share/icons/hicolor/scalable/apps/rs.ruffle.Ruffle.svg
+      - install -D -T ./desktop/packages/linux/rs.ruffle.Ruffle.desktop /app/share/applications/rs.ruffle.Ruffle.desktop
+      - install -D -T ./desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml /app/share/metainfo/rs.ruffle.Ruffle.metainfo.xml
+      - install -D -T ./target/release/ruffle_desktop /app/bin/ruffle
+    sources:
+      - type: git
+        url: https://github.com/ruffle-rs/ruffle.git
+        tag: nightly-2024-08-04
```

Comment:
Is there a stable tag for this? Or do ruffle only publishes nightly tags? 

https://docs.flathub.org/docs/for-app-authors/requirements#stable-releases

Reply:
We are in the process of releasing Ruffle 1.0.0, currently we have only nightly releases, but we do not plan to update flatpak daily.

When 1.0.0 gets released (and it should be soon), I will switch this to the stable version tag and remove `patches/release.patch`, as we will probably be updating our metainfo with releases at that point.

Is it a blocker or is it acceptable to merge and switch to stable later?

Reply:
I think as long as this doesn't get updated daily, it is probably fine. You should not enable external-data-checker (x-checker) too as the PRs will pile up due to high release frequency. It runs hourly, daily.

You can update it weekly or semi-weekly etc.

Reply:
Alright! I'll set up the checker when we have stable releases (we will probably be releasing stable ~monthly).